arbitrum-docs/faqs/gas-faqs.mdx

```diff
@@ -0,0 +1,33 @@
+# FAQs: Gas
+
+### How does gas work on Arbitrum chains? {#q-gas}
+
+Fees on Arbitrum chains are collected on L2 in the chains' native currency (ETH on both Arbitrum One and Nova).
+
+A transaction fee is comprised of both an L1 and an L2 component:
+
+The L1 component is meant to compensate the Sequencer for the cost of posting transactions on L1 (but no more). (See [L1 Pricing](./arbos/l1-pricing.mdx).)
+
+The L2 component covers the cost of operating the L2 chain; it uses Geth for gas calculation and thus behaves nearly identically to L1 Ethereum (See [Gas](./arbos/gas.mdx)).
+
+L2 Gas price adjusts responsively to chain congestion, ala EIP 1559.
+
+### Why does it look like two identical transactions consume a different amount of gas? {#q-gas-changes}
+
+Calling an Arbitrum node's `eth_estimateGas` RPC returns a value sufficient to cover both the L1 and L2 components of the fee for the current gas price; this is the value that, e.g., will appear in users' wallets in the "gas limit" field.
+
+Thus, if the L1 calldata price changes over time, it will appear (in e.g., a wallet) that a transaction's gas limit is changing. In fact, the L2 gas limit isn't changing, merely the total gas required to cover the transaction's L1 + L2 fees.
+
+See [2-D fees](https://medium.com/offchainlabs/understanding-arbitrum-2-dimensional-fees-fd1d582596c9) for more.
+
+### How is the L1 portion of an Arbitrum transaction’s gas fee computed? {#q-gas-l1}
+
+The L1 fee that a transaction is required to pay is determined by compressing its data with brotli and multiplying the size of the result (in bytes) by ArbOS’s current call data price; the later value can be queried via the `getPricesInWei` method of the `ArbGasInfo` precompile.
+
+### Does Arbitrum have a mempool? {#q-mempool}
```

Comment:
Should we add the fact that this queue is empty most of the time as txs are sequenced immediately?